src/styles/select.scss

```diff
@@ -131,7 +131,7 @@ $outline-offset: 0.1875rem;
 
     font-size: var(--sapFontSize);
     padding-left: $fd-select-padding-x;
-    min-width: 5.625rem;
+    min-width: 2.125rem;
```

Comment:
how did you come up with this value? the specs says 5rem min-width.

Reply:
I guessed that `5rem` it's the `min-width` for the whole select. I came up with this value by calculating `5rem - border - button width`. So the with of the select is `5rem`\`80px` as shown on the screenshot.

Reply:
We need to ask designers about this. I don't think we can guess on this. I will send an email and cc you.

Reply:
Okay, thanks. **[BLOCKED]** till getting the answer from designers.